Samples/Package/Out-of-Box/StoreAPP-OOB-Sample/scripts/outofbox/close.ps1

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+push-location $PSScriptRoot
```

Comment:
>push-location [](http://example.com/codeflow?start=0&length=13)

TODO in next PR: standarlize the PowerShell cmdlets to capilize first letter of each word as documented